index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <!-- adding comments to check code from here -->
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+    <!-- check html elements arrangment -->
+    <head><title></title>
+    <link rel="stylesheet" type="text/css" href="css/style.css"/></head>
+    <body>
+      <nav id="navbar" style="position:sticky;top:0;">
+        <a href="#welcome-section">Welcome Section</a>
+        </nav>
+      <style type="text/css">
+    @media only screen and (max-width: 600px)
+    {
+    #welcome-section {
+        color:#ffffff;
+      }
+    }
+      </style>
+    <!-- check this section as well for review -->
+    <section id="welcome-section">
+    <h1>Hi! I am a Software Developer</h1>
+      </section>
+      <!-- review this section as well -->
+      <section id="projects">
+      <a id="profile-link" class="project-tile" href="" target="_blank"><img class="project-image" src="https://cdn.freecodecamp.org/testable-projects-fcc/images/tribute.jpg"
+    "title="project"></img>
```

Comment:
Please configure linters ,is a tool that analyzes your source code to flag programming errors, bugs, stylistic errors, and suspicious constructs